packages/flutter/lib/src/rendering/debug_overflow_indicator.dart

```diff
@@ -248,8 +247,8 @@ mixin DebugOverflowIndicatorMixin on RenderObject {
         context: ErrorDescription('during layout'),
         informationCollector: () sync* {
           if (debugCreator != null)
-            yield DiagnosticsDebugCreator(debugCreator);
-          yield* overflowHints;
+            yield DiagnosticsDebugCreator(debugCreator as Object);
```

Comment:
In my PR I changed "debugCreator" to be `Object? debugCreator` so this becomes `debugCreator!`, it's probably an improvement to the API but it does require some changes elsewhere

Reply:
Done